transcom/mymove
# B 18840 int

## [B-18840](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a907285)

## Summary

Adds additional columns to the mto_shipments table for pro gear as requested in B-18840 as an upstream enabler to later work.

Is there anything you would like reviewers to give additional scrutiny?
Please ensure database migrations are functional.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the database as a developer using a Postgres client tool
2. View the mto_shipments table
3. Ensure the two columns actual_pro_gear_weight(INTEGER, NULLABLE) and actual_spouse_pro_gear_weight(INTEGER, NULLABLE) exist. Note: As an enabler, more functional testing will come with downstream tasks that leverage these columns.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
